### PR TITLE
Husky Observer Documentation

### PR DESCRIPTION
**Pull request by civerachb-cpr**
Husky Observer docs ready for review and to be merged into development, and then eventually info the main release branch.

### CODE REVIEW

File: docs_versioned_docs/version-ros1noetic/robots/solutions/husky_observer/troubleshooting_husky_observer.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
```
Comment (jmastrangelo-cpr) [inline docs_versioned_docs/version-ros1noetic/robots/solutions/husky_observer/troubleshooting_husky_observer.mdx:1]: should we remove all of this and link to the updated outdoornav FAQs

File: docs_outdoornav_user_manual/api/api_endpoints/definitions.mdx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/api/api_endpoints/definitions.mdx:1]: @civerachb-cpr  Should all the changes in these docs_outdoornav_user_manual files be included in the docs for the 0.9.0 release? If so, just want to make sure we get this all merged in prior to @dobrienCP creating the 0.9 release for the OutdoorNav docs.

File: docs_versioned_docs/version-ros1noetic/robots/solutions/husky_observer/troubleshooting_husky_observer.mdx (hunk 1 above)
Comment (jhiggins-cpr) [reply docs_versioned_docs/version-ros1noetic/robots/solutions/husky_observer/troubleshooting_husky_observer.mdx:1]: Agreed. Updated.

File: docs_outdoornav_user_manual/api/api_endpoints/definitions.mdx (hunk 2 above)
Comment (jhiggins-cpr) [reply docs_outdoornav_user_manual/api/api_endpoints/definitions.mdx:1]: Discussed offline with Chris and confirmed that we want these changes.

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**